test/test_wordcloud.py

```diff
@@ -352,8 +375,8 @@ def test_recolor_too_small_set_default():
 def test_small_canvas():
     # check font size fallback works on small canvas
     wc = WordCloud(max_words=50, width=21, height=21)
-    wc.generate(THIS)
-    assert len(wc.layout_) == 1
+    wc.generate(SMALL_CANVAS)
```

Comment:
Is the SMALL_CANVAS change related or from a rebase?

Reply:
I'm not familiar with the rebase.  I did make changes to `SMALL_CANVAS` because this test didn't pass, even though my changes seem irrelevant to it.  We discussed this issue [here](https://github.com/amueller/word_cloud/pull/529#issuecomment-592735274)

Reply:
> from a rebase?

@amueller You can see the difference between the before and after rebase clicking on "force-pushed" link:

![image](https://user-images.githubusercontent.com/219043/78847272-4d028c00-79dc-11ea-9ff4-30fc38dc940f.png)

I this case, the differences are visible using this link: https://github.com/amueller/word_cloud/compare/bacd96e25e083b57378a4363746286dda32803a8..3983ef8056fb0254d2a6365811a09d3efe75fc43

Reply:
The error should have been fixed in #525